src/main/java/org/broadinstitute/hellbender/tools/walkers/varianteval/AlleleFrequencyQC.java

```diff
@@ -59,15 +60,8 @@ public void onTraversalStart() {
         metricOutput = outFile; // output file for summary metrics
 
         // have to set the output file for variant eval; if not given a debug file to return the variant eval results
-        // from, this will just be a temporary file that wil lbe deleted after the tool runs
-        try {
-            outFile = debugFile == null ? File.createTempFile("variant_eval" ,".txt") : debugFile;
```

Comment:
Should we worry about this no longer catching the exception? Now this will err on this line `throw new GATKException("Cannot create temp file: " + ex.getMessage(), ex);` which seems less specific but probably equally as informative? This is probably fine?